library/objective-c/EnvoyConfiguration.m

```diff
@@ -16,6 +16,7 @@ - (instancetype)initWithAdminInterfaceEnabled:(BOOL)adminInterfaceEnabled
                               enableHappyEyeballs:(BOOL)enableHappyEyeballs
                            enableInterfaceBinding:(BOOL)enableInterfaceBinding
                     enforceTrustChainVerification:(BOOL)enforceTrustChainVerification
+                enableDNSFilterUnroutableFamilies:(BOOL)enableDNSFilterUnroutableFamilies
```

Comment:
nit: how about `filterUnroutableDNSResults` (or perhaps, `filterUnroutableDNSResultIPs`)?

Reply:
`enableAppleDNSFiltering`

Reply:
Filter is ambiguous if we're keeping or rejecting unroutable results.

Perhaps `excludeUnroutableDNSResults` since we can share that setting across platforms since we also have the option to do that on the c-ares side.